File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q21.js
import { question } from "readline-sync";
function main(){
    //Input
    const temperatureF = parseFloat(question("Enter a temperature in °F: "))

    //Processing
    const temperatureC = ((5 * temperatureF) - 160)/9

    //Output
    console.log(`The temperature of ${temperatureF} °F converted in °C is equal to: ${temperatureC} °C`)

}
main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q21.py
def main():
    
    #Input
    temperatureF = float(input("Enter a temperature in °F: "))

    #Processing
    temperatureC = ((5 * temperatureF - 160)/9)
    rounded_temperatureC = round(temperatureC, 2)

    #Output
    print(f"The temperature of {temperatureF} °F converted to °C is equal to: {rounded_temperatureC}°C ")

main()